examples/metrics/Dockerfile

```diff
@@ -30,11 +32,13 @@ RUN echo '{\n\
 # ---
 # Stage 2: Final image with nothing but certs, binary, and default config file
 # ---
-FROM scratch AS final
+FROM gcr.io/distroless/base AS final
+ARG OS
+ARG ARCH
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
-COPY --from=builder /go/src/github.com/project-zot/zot/bin/zot /zot
```

Comment:
I think what we need to do is to keep this file similar to Dockerfile-conformance from which this file was originally inspired from (maybe rename Dockerfile -> Dockerfile-conformance ?)

I'm thinking of changing this line to:
`COPY --from=builder /go/src/github.com/project-zot/zot/bin/zot-linux-amd64 /usr/bin/zot`
There ios no need for OS & ARCH args as the kind-setup.sh script was intended to be run on linux systems and was not tested on other platforms.